### ISSUE TITLE
Number of entries in clipboard setting

### ISSUE THREAD

**Issue by Perelandra0x309** [humdingerb/clipdinger#11]
First, congratulations on making it to 1.0!

In the settings window if you decrease the value of the "entries in the clipboard history" you actually get value + 1 entries.  But if you increase from the current value you get the correct number.  Try starting at 4, then 3, then 2, then 1.  At each change Press "OK" to save the settings.  You can copy text each time you change the value and there will still be value +1 entries.  Then increase to 2, 3, 4 and again copy some text each time and there are the correct number of entries.

**Comment by humdingerb** [comment]
Thanks, "making it to 1.0" was easy. I just called it v1.0. :)
Seriously, with the drag&drop support it now has everything I imagined for it. So new versions will probably only be bug fixes, I expect.

So, thanks for reporting! You're right, never noticed it. Normally you the history is dozens of entries deep, and generally, having one more or less doesn't matter that much... :) Nevertheless, it's fixed. As is a history of "0", which isn't possible and changes to "1".

BTW, I know that it's a bit of a misnomer, because the number of entires always counts the current contents of the clipboard at the top of the list to the history. But I deem it's less puzzling that to enter a 5 in the the settings and see a max of six entries in the list (1 contents + 5 history entries).
